app/views/alaveteli_pro/info_request_batches/_authority_list.text.erb

```diff
@@ -0,0 +1,11 @@
+<% if public_bodies.size %>
+<% example_body_names = public_bodies.take(2).map(&:name) %>
+<%= n_('{{count}} recipient, {{first_public_body}}',
```

Comment:
Ah - so I just read through https://github.com/mysociety/alaveteli/pull/3443 and realised what's going on here. I initially assumed that rails wouldn't escape this because it was in a text file (due to https://github.com/rails/rails/pull/8235), but then I saw that it was when I tested it in the console. So I *think* this is actually ok, thanks to the magic in our translation helpers: https://github.com/mysociety/alaveteli/blob/develop/lib/i18n_fixes.rb#L39

*However* when we use this in a text email, we'll actually want to mark the body names as `html_safe` so that they *don't* get escaped (along with a variety of other things we pass into translation strings like the site name, law name, etc). Given that, I think I should make the text and html versions of this partial separate, and explicitly mark these body names as html_safe in the text one.